src/components/Announcement/AnnoucementItem.tsx

```diff
@@ -0,0 +1,109 @@
+import styled, { CSSProperties } from 'styled-components'
+
+import NotificationImage from 'assets/images/notification_default.png'
+import { formatTime, useNavigateCtaPopup } from 'components/Announcement/helper'
+import { Announcement } from 'components/Announcement/type'
+import Column from 'components/Column'
+
+const HEIGHT = '100px'
+
+const Wrapper = styled.div`
+  border-bottom: 1px solid ${({ theme }) => theme.border};
+  background-color: ${({ theme }) => theme.background};
+  font-size: 12px;
+  padding: 16px 20px;
+  gap: 14px;
+  display: flex;
+  align-items: flex-start;
+  cursor: pointer;
+  :hover {
+    background-color: ${({ theme }) => theme.buttonBlack};
+  }
+`
+
+const Title = styled.div`
+  font-size: 12px;
+  font-weight: 500;
+  color: ${({ theme }) => theme.text};
+  height: 32px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
+`
+
+const Desc = styled.div`
+  font-size: 12px;
+  color: ${({ theme }) => theme.subText};
+  height: 34px;
+
+  display: block;
+  display: -webkit-box;
+  max-width: 100%;
+  line-height: 16px;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  > p {
+    margin: 0;
+  }
+`
+const Time = styled.span`
+  color: ${({ theme }) => theme.border};
+  text-align: right;
+  width: 100%;
+`
+const RowItem = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 6px;
+  align-items: flex-start;
+  flex: 1;
+  justify-content: space-between;
+  overflow: hidden;
+  height: ${HEIGHT};
+`
+
+const Image = styled.img`
+  width: ${HEIGHT};
+  max-height: ${HEIGHT};
+  border-radius: 8px;
+  object-fit: contain;
```

Comment:
consider to use object-fit: scale-down, I see some images that lower than 100px is magnified, make it losing detail
![image](https://user-images.githubusercontent.com/9592090/220517934-7bbaf3da-8993-48de-ac34-660a809c78fc.png)

Reply:
also I see a bug of the TopBanner 
![image](https://user-images.githubusercontent.com/9592090/220519176-3c4ae440-506e-4a19-b757-b4edb09946b2.png)

Reply:
![image](https://user-images.githubusercontent.com/9592090/220519548-cad49109-97b7-436e-a15c-16e191047f5d.png)
icons not centered, See more position looks weird

Reply:
- arrow shape: this is behavior of `swiper`
- bell icon image from design
- See more see less/ see less: I will give feedback to the designer.

Reply:
there should be someway to customize the arrows, it look really bad for production, also there is bug of border-radius while swiping